internal/cli/default_setter_opts.go

```diff
@@ -230,6 +239,7 @@ func (opts *DefaultSetterOpts) AskOrg() error {
 		if err2 := telemetry.TrackAskOne(p, &manually); err2 != nil {
 			return err2
 		}
+		opts.AskedOrgsOrProjects = true
```

Comment:
why are we setting AskedOrgsOrProjects  to true here?

Reply:
We should also count `Do you want to enter the Org ID manually?` as asked. Trying to track if any reference to orgs and projects was done here.